### PR TITLE
Avoid to setup provisioners in a fully containerized environment

### PR DESCRIPTION
**Pull request by fmount**
This commit adds a when clause to avoid the setup of grafana
provisioners in a fully containerized scenario.
This is needed when the ceph-grafana-dashboards package is not
installed and this task could result in a wrong grafana
configuration that let the container crash.

Signed-off-by: fmount <fpantano@redhat.com>

### CODE REVIEW

**Comment by fmount** [comment]
According to the https://github.com/ceph/ceph/pull/26415 in a fully containerized scenario we need to populate the grafana provisioners in a different way (injecting the layouts using the api), so we need to skip the task.

File: roles/ceph-grafana/tasks/configure_grafana.yml [hunk 1]
```diff
@@ -46,6 +47,7 @@
     src: dashboards-ceph-dashboard.yml.j2
     dest: /etc/grafana/provisioning/dashboards/ceph-dashboard.yml
     mode: 0640
+  when: not containerized_deployment
```
Comment (dsavineau) [inline roles/ceph-grafana/tasks/configure_grafana.yml]: could you add the bool filter there ? ( `| bool` )

Comment (fmount) [reply roles/ceph-grafana/tasks/configure_grafana.yml]: done

**Comment by guits** [comment]
jenkins test pipeline

**Comment by gfidente** [comment]
nice, I guess we want the backport-4.0 label on this one?

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
nfs-ganesha dev repositorie failures, merging anyway

**Pull request merged**